pkg/network/tags.go

```diff
@@ -0,0 +1,48 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+package network
+
+// ConnTag contain a bitfileds of tags
+type ConnTag = uint64
```

Comment:
I think this is OK for now. We can improve this later. I think this is tricky because the `http` package cannot import from `pkg/network`, otherwise it creates an import loop.

Reply:
I removed it for the moment, as we don't need it